packages/manager/cypress/support/login.ts

```diff
@@ -9,7 +9,7 @@ import { apiMatcher } from 'support/util/intercepts';
 
 const overrideLocalStorage = (
   window: any,
```

Comment:
while you're here

```suggestion
  window: Window,
```

same with `_loginWithToken(win)` and `onBeforeLoad: (win: any) => {`